.yarnrc.yml

```diff
@@ -2,14 +2,15 @@ defaultSemverRangePrefix: ""
 
 enableGlobalCache: false
 
-nmMode: hardlinks-global
-
-nodeLinker: node-modules
+nodeLinker: pnpm
 
 packageExtensions:
   "@endemolshinegroup/cosmiconfig-typescript-loader@*":
     peerDependencies:
       typescript: ^4.5.2
+  "next-secure-headers@*":
```

Comment:
To fix missing dep

See https://github.com/jagaapple/next-secure-headers/pull/54